### DISCUSSION TITLE
Prediction of Mutational Effects

### DISCUSSION THREAD

**Discussion by EBGU** [facebookresearch/esm#288] [Q&A]
Hi!
I wonder where I can download the full dataset for the prediction of mutation effects (as in the paper:_Collectively, the data includes over 700,000 variant effect measurements from over 100 large-scale experimental mutagenesis datasets._).Thank you very much!

**Comment by tomsercu** [comment]
You can find our predictions as well as the baselines' predictions in the last section of this README https://github.com/facebookresearch/esm/tree/main/examples/variant-prediction#per-task-performance
